src/components/CalendarDay.jsx

```diff
@@ -16,7 +16,7 @@ const propTypes = forbidExtraProps({
   day: momentPropTypes.momentObj,
   daySize: nonNegativeInteger,
   isOutsideDay: PropTypes.bool,
-  modifiers: PropTypes.object,
+  modifiers: PropTypes.instanceOf(Set),
```

Comment:
This PR is already semver-major due to the fact that the `DayPicker` now functions totally differently. Do you think this is a concern? I kind of think this project was not working with IE8 already.